python/CreatePipWhl.cmake

```diff
@@ -31,7 +31,15 @@ if (PYTHON)
     file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/python/ngraph_bridge)
 
     # Get the list of libraries we need for the Python pip package
-    file(GLOB NGRAPH_LIB_FILES "${NGTF_INSTALL_DIR}/lib/lib*")
+    # If we are building on CentOS then it's lib64 - else lib
+    set(LIB_SUFFIX lib)
+    if(NOT APPLE)
+        if(${OS_VERSION} STREQUAL "centos")
```

Comment:
I tried running ngraph-packaging with this ngraph-tf branch on gcc4.8.  I encountered an error:

-- CMAKE_CURRENT_SOURCE_DIR: /home/ngraph-packaging/ngraph-tf/python
-- CMAKE_CURRENT_BINARY_DIR: /home/ngraph-packaging/ngraph-tf/build
CMake Error at /home/ngraph-packaging/ngraph-tf/python/CreatePipWhl.cmake:37 (if):
  if given arguments:

    "STREQUAL" "centos"

  Unknown arguments specified
Call Stack (most recent call first):
  src/cmake_install.cmake:100 (include)
  cmake_install.cmake:39 (include)
  
Makefile:61: recipe for target 'install' failed

Reply:
Fixed.